code/game/atoms.dm

```diff
@@ -1341,3 +1341,7 @@
 				max_grav = max(G.setting,max_grav)
 			return max_grav
 	return SSmapping.level_trait(T.z, ZTRAIT_GRAVITY)
+
+///Called when something resists while this atom is its loc
+/atom/proc/resist_act(mob/living/user)

```

Comment:
`container_resist_act` would make more sense here, since resisting is a more generic concept.